laravel/app/Models/Resident.php

```diff
@@ -19,4 +19,19 @@ class Resident extends Model
      * @var bool
      */
     public $timestamps = false;
+
```

Comment:
missing resident -> probability relationship (I think it's one to one?)
missing resident -> rotation relationship (I think it's one to many?)